commons/libs/constants/Environments.ts

```diff
@@ -1,3 +1,3 @@
-type Environments = 'local' | 'dev' | 'qa' | 'production' | 'test'
+enum Environments { local = 'local', dev = 'dev', qa = 'qa', production = 'production', test = 'test' }
```

Comment:
should these be uppercase

Reply:
converted to upper case.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/947aff4e9347b40cd886869abb46644aa18e1dcb)